best-practices/README.md

```diff
@@ -101,6 +101,10 @@ Rails
   environment variables are detected on deploy.
 * [Use blocks][date-block] when declaring date and time attributes in FactoryGirl factories.
 * Use `touch: true` when declaring `belongs_to` relationships.
+* Name service classes after the action they perform
```

Comment:
In the projects I have been on, we've put method objects in `app/models` if they are a part of the applications business logic or `app/lib` if they're not.